packages/cli/src/controllers/oauth/oAuth1Credential.controller.ts

```diff
@@ -33,7 +33,7 @@ export class OAuth1CredentialController extends AbstractOAuthController {
 	@Get('/auth')
 	async getAuthUri(req: OAuthRequest.OAuth1Credential.Auth): Promise<string> {
```

Comment:
Is the type here correct if `req.user` was not caught by the type system? Same question to the other endpoint handlers

Reply:
I think I do not understandd how `never` works. 
`AuthlessRequest` had `user: never` and somehow it was not a type error.
I've removed `user` from `AuthlessRequest`, and now `req.user` is a type-error in this context.

![Screenshot from 2024-07-31 16-29-49](https://github.com/user-attachments/assets/3023f0dd-15a8-453e-bd91-51de9372c8f5)